src/common/m_variables_conversion.fpp

```diff
@@ -1709,4 +1710,118 @@ contains
     end subroutine s_compute_fast_magnetosonic_speed
 #endif
 
+#ifndef MFC_PRE_PROCESS
+    subroutine s_compute_wave_speed(wave_speeds, vel_L, vel_R, pres_L, pres_R, rho_L, rho_R, rho_avg, &
```

Comment:
surely some of these arguments should be optional since not every riemann solver uses them all?